src/parcsr_ls/gen_redcs_mat.c

```diff
@@ -303,7 +303,7 @@ hypre_seqAMGSetup( hypre_ParAMGData *amg_data,
 
             F_seq = hypre_ParVectorCreate(seq_comm, size, row_starts);
             U_seq = hypre_ParVectorCreate(seq_comm, size, row_starts);
-            hypre_ParVectorOwnsPartitioning(F_seq) = 0;
```

Comment:
Why `ParVector` still keep this information about who owns the partitioning?

Reply:
I agree with changing this as mentioned in the comment bellow